### ISSUE TITLE
Installing several data plane fails

### ISSUE THREAD

**Issue by enlight3d** [l7mp/stunner-helm#16]
Hello, I'm trying to install a second data plane for test purposes and I'm running into an issue. I just followed the documentation example to deploy a simple data plane with the helm chart without changing any values. This is the output I'm getting when trying to install it :
```
helm install --namespace=stunner --timeout=10m0s --values=/home/shell/helm/values-stunner-0.15.0.yaml --version=0.15.0 --wait=true stunner /home/shell/helm/stunner-0.15.0.tgz 2023-07-21T12:16:25.432166310Z Error: INSTALLATION FAILED: rendered manifests contain a resource that already exists. Unable to continue with install: ClusterRole "stunner-config-watcher-clusterrole" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-name" must equal "stunner": current value is "stunner-0-1688051195"; annotation validation error: key "meta.helm.sh/release-namespace" must equal "stunner": current value is "stunner-xxx-dev"
```

FYI, I have a running data plane in another namespace called "stunner-xxx-dev".

Let me know if you need other infos from me !

My k8s cluster is running kubernetes v1.24.13 and I'm using Rancher v2.6.8

**Comment by davidkornel** [comment]
Hi @enlight3d 

Excuse me for the delay somehow I missed this. I see and understand your problem. Not so long ago we introduced a configwatcher sidecar in the stunner pods that watches the configmap and refreshes stunnerd.conf immediately without any delay. For that, this clusterrole was needed, and obviously, since that resource is not namespaced it exists already. 

I realized now that there is no reason to have it as a ClusterRole, a Role is enough since we only want to watch configmaps in the same namespace and not cluster-wide.

I will notify you when it's fixed
